drivers/usb/udc/udc_numaker.c

```diff
@@ -172,7 +172,7 @@ static inline void numaker_usbd_sw_connect(const struct device *dev)
 
 	/* Enable relevant interrupts */
 	base->INTEN = USBD_INT_BUS | USBD_INT_USB | USBD_INT_FLDET |
-		      IF_ENABLED(CONFIG_UDC_ENABLE_SOF, (USBD_INT_SOF |))
+		      IF_ENABLED(CONFIG_UDC_ENABLE_SOF, (USBD_INT_SOF |)) /* High CPU load */
```

Comment:
"High CPU load" comment is a bit extreme. It depends on the bus speed, the CPU, and the tasks that need to be performed during the SOF event. However, that is not relevant here.